crates/flashblocks-rpc/README.md

```diff
@@ -17,10 +17,10 @@ cargo build --release
 Run:
 
 ```bash
-./target/release/flashblocks-rpc \
-    --flashblocks.enabled=true \
-    --flashblocks.websocket-url=ws://localhost:8080/flashblocks \
-    --chain=optimism \
+./target/release/flashblocks-rpc node \
+    --flashblocks.enabled \
+    --flashblocks.websocket-url="wss://sepolia-flashblocks.unichain.org/ws" \
```

Comment:
would prefer to have the local node here instead of pointing to unichain in the readme